### DISCUSSION TITLE
How to delete an organization?

### DISCUSSION THREAD

**Discussion by henry-bao** [planetscale/discussion#449] [Q&A] [answered]
Hello everyone, recently I've been exploring the PlanetScale dashboard and created a new organization for testing purposes. I assumed it would be straightforward to delete the organization afterwards, but I can't seem to find a method other than deleting the entire account. My apologies if I missed an easy solution. Appreciate your help!

**Comment by CanRau** [comment]
Same here, in my case iirc an org has been auto created when I invited my work mail which I'd like to delete to not confuse myself 😅

**Comment by fooforge** [comment, marked as answer]
@henry-bao, @CanRau We have that on our radar to implement, but for now as a user you can't delete an organization. We can delete them for you, however. Make sure the to-be-deleted organizations do not have any databases associated with it anymore before you [open a ticket](http://support.planetscale.com) with us, and we'll get these squared away for you shortly.
